### PR TITLE
Newui

### PR DESCRIPTION
**Pull request by mahtabsabet**
Changed the UI entirely - the event listing page, as well as all the other views to conform with the new UI.

### CODE REVIEW

File: app/controllers/ListEventsController.js [hunk 1]
```diff
@@ -9,10 +10,21 @@ GameUpApp.controller('ListEventsController', ['$scope', '$location', function($s
     return query.find();
   };
 
+  $scope.retrievePromotedEventList = function(){
```
Comment (tylerhaugen-stanley) [inline app/controllers/ListEventsController.js:13]: What happens when there is 20 promoted events lets say, and only 15 spots for promoted events? Would this be a job for the controller to handle this logic?

Comment (mahtabsabet) [inline app/controllers/ListEventsController.js:13]: Good point. I guess we should add a limit to the number of promoted events that are listed. Hopefully though the total number of promoted events wouldn't ever be more than 5 or something.

We'd also have to think about promoted events by region - the promoted events shown in Toronto will be different for the ones in Ottawa. I don't know how we'd work out that logic...

Comment (ajitpawar) [inline app/controllers/ListEventsController.js:13]: You can set limits on your Parse query
`query.limit(10);`
If you don't explicitly set a limit, the default is 100, I think.

Also, fetching events by region could be done as follows:

```
var point = new Parse.GeoPoint({latitude: 40.0, longitude: -30.0});
query.near("location", point);
```

We can set the "location" attribute to Toronto's latitude & longitude by default

Comment (joeyfreund) [inline app/controllers/ListEventsController.js:13]: +1 @ajitpawar 

Also, I would consider defining the `retrievePromotedEventList` as follows:

``` js
$scope.retrievePromotedEventList = function(options){
    // ...
```

Where `options` is an object that specifies the query parameters.  
For example:

``` js
options = {
    limit: 5,
    location: {lat: 40.0, lon: -30.0}
} 
```

**Pull request merged**